scripts/docker/run.build-nilrt.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Why the first '.' in `run.build-nilrt.sh`? Is that some specific file name format you are following?

Reply:
Nope; that's just how I like to name stuff.